unaligned_bam_to_bqsr_and_vcf/bwa_mem.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "run bwa_mem"
+baseCommand: ["/usr/local/bin/bwa", "mem"]
+arguments: 
+    ["-K", "10000000",
+    "-t", "8",
```

Comment:
Just to leave this comment in a more official place... this could be:
```yaml
    - { valueFrom: $(runtime.cores) }
```
combined with a:
```yaml
    ResourceRequirement:
        coresMin: 8
```
in the requirements section.